omod/src/test/java/org/openmrs/module/cohort/web/resource/CohortResourceTest.java

```diff
@@ -41,6 +51,20 @@ public void setup() {
 		cohort = new CohortM();
 		cohort.setUuid(COHORT_UUID);
 		cohort.setName(COHORT_NAME);
+		CohortAttributeType attributeType = new CohortAttributeType();
```

Comment:
So, the test added here doesn't really test anything useful. That's because the method it uses is really just trying to verify that the endpoint correctly calls `cohortService.getCohort()`. That the attributes are returned are just a result of the fact that we've told the mock to return the cohort object which, in our setup, has the attributes attached. That's not really testing anything.

What we need is a test that calls the create or update end-point to add an attribute to the user and ensures that those attributes are returned.